R/condition.R

```diff
@@ -13,6 +16,11 @@
 #'   whether or not the expression is the correct answer or not, \code{correct},
 #'   the type of expression (formula, function, or value), \code{type}
 #' @export
+#'
+#' @examples
+#' \dontrun{
```

Comment:
is there a reason for the `\dontrun`?

Reply:
nope, I thought it would evaluate `.result` (which it doesn't)

Reply:
remove the `\dontrun`